experimental/LieAlgebras/src/PathModel.jl

```diff
@@ -0,0 +1,617 @@
+##### Kashiwara Crystal #####
+
+abstract type AbstractCrystal <: AbstractAlgebra.Set end
+
+abstract type AbstractCrystalElem <: AbstractAlgebra.SetElem end
+
+# AbstractCrystalElem required methods
+
+function Base.eps(b::AbstractCrystalElem)
```

Comment:
please don't overload the julia base function that gives you the machine precision